packages/libs/react-ui/src/components/Pagination/Pagination.stories.tsx

```diff
@@ -14,11 +14,20 @@ const meta: Meta<
   } & IPaginationProps
 > = {
   title: 'Components/Pagination',
+  parameters: {
+    docs: {
+      description: {
+        component:
+          '<i>Note: maximum navigation items is currently limited (not technically enforced).</i><br><br><strong>Label </strong><br> is used to allow screen readers to notify user that there is a pagination here.',
```

Comment:
What do you mean by this? `<i>Note: maximum navigation items is currently limited (not technically enforced)`

Reply:
I took that from the NavHeader, but I thought it would be nice to notify the user that the limit we put in the storybook for the total number of pages is only for the usage of the component here (it's not a technical limitation)
![Screenshot 2023-08-24 at 16 33 55](https://github.com/kadena-community/kadena.js/assets/132443209/6fd612d6-21b1-4bc1-a2a2-f8bb1594bde6)

Reply:
The label description is duplicated in the props section. I do think that this text overall is more confusing than it is helpful right now. I would suggest just writing a a plain text description for what the component does/is used for